.github/ISSUE_TEMPLATE/inconsistency.md

```diff
@@ -0,0 +1,33 @@
+# Inconsistency
+
+This template contains an example inconsistency report. Please replace all text except for the checklist and the section headers (they start with \#). Thank you for your contribution to improving ESI.
+
+Please provide a short description of the inconsistency you have noticed. E.g.:
+
+Sometimes the corporation ID is returned as `id` and other times as `corporation_id`
+
+## Routes
+
+List the routes (with version and method) that are affected by this inconsistency. Split into groups if appropriate. E.g.:
+
+Corporation ID as `corporation_id`:
+
+- `GET /v1/corporations/{corporation_id}/`
+- all other corporation routes
+
+Corporation ID as `id`:
+
+- `GET /v1/characters/{character_id}/medals/`
+
+## Resolution
+
+Propose a solution to resolve the inconsistency. E.g.:
+
+Change the `id` attribute in the return of `GET /v1/characters/{character_id}/medals/` to `corporation_id`
+
+# Checklist
+
+- [ ] Inconsistency is documented
+- [ ] Inconsistent route(s) are documented
+- [ ] Resolution will require a version increase
```

Comment:
increase -> bump.

Reply:
Also supply a link to [breaking_changes.md](https://github.com/ccpgames/esi-issues/blob/master/breaking_changes.md) as a reference for this and the checkbox below this.